components/confirm_modal.jsx

```diff
@@ -175,6 +175,7 @@ export default class ConfirmModal extends React.Component {
                 <Modal.Footer>
                     {cancelButton}
                     <button
+                        autoFocus='true'
```

Comment:
This is a site-wide change, right? Have we confirmed we want this behaviour? What's this fixing, precisely?

Reply:
was requested to get the autofocus

Reply:
I think we should tease this and the `keydown` behaviour out of this PR and submit it separately. My concern is that, in the event of unexpected side effect, this PR doesn't "stand out" as a cause unless one looks deep into the commits. Even then, it would be annoying to revert the whole thing instead of just the orthogonal changes here.

Reply:
ok I will remove this and submit in another PR

Reply:
done

Reply:
will open another PR only with this change

Reply:
0/5, but wondering if we should push `autoFocus` too. Otherwise I've approved the PR :)